### ISSUE TITLE
DB - wont process the reservation (Liz)

### ISSUE THREAD

**Issue by ronenreinheimer** [206Tours/DB#942]
![image](https://cloud.githubusercontent.com/assets/10079829/13816297/24479446-eb64-11e5-91f8-41383b61ff26.png)
![image](https://cloud.githubusercontent.com/assets/10079829/13816359/70317340-eb64-11e5-965e-dd633918f8a4.png)

**Comment by ronenreinheimer** [comment]
Liz says that she plugged in all the CC info and checked terms and conditions and pressed confirm order – the page loaded but went to a blank screen. Do you want us to send you the credit card info? Liz checked with Ali and she doesn’t see a payment in the system.

**Comment by ronenreinheimer** [comment]
OK, we are doing the booking together and i am taking screen shots. we noticed that the insurance is not being added to the deposit amount. I pressed the confirm order button and it did not process the order. i took video and will email to you.

![image](https://cloud.githubusercontent.com/assets/10079829/13819505/19ec4ff8-eb70-11e5-8dca-39118c7a9fa3.png)

![image](https://cloud.githubusercontent.com/assets/10079829/13819573/5a56d252-eb70-11e5-802d-031a98483635.png)
